sympy/functions/elementary/tests/test_integers.py

```diff
@@ -126,6 +131,70 @@ def test_floor():
     assert Eq(floor(y), y - frac(y))
     assert Eq(floor(y), -ceiling(-y))
 
+    neg = Symbol('neg', negative=True)
+    nn = Symbol('nn', nonnegative=True)
+    pos = Symbol('pos', positive=True)
+    np = Symbol('np', nonpositive=True)
+
+    assert (floor(neg) < 0) == True
+    assert (floor(neg) <= 0) == True
+    assert (floor(neg) > 0) == False
+    assert (floor(neg) >= 0) == False
+    assert (floor(neg) <= -1) == True
+    assert (floor(neg) >= -3) == (neg >= -3)
+    assert (floor(neg) < 5) == (neg < 5)
```

Comment:
Should this not be True?

Reply:
I just tested with this PR and (neg < 5) evaluates to True so it would be best to change the test here.

Reply:
It's not a big deal though and this looks good so I'll merge.